examples/helm/templates/jicofo/xmpp-secret.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "call-nested" (list . "prosody" "prosody.fullname") }}-jicofo
+  labels:
+    {{- include "jitsi-meet.jicofo.labels" . | nindent 4 }}
+type: Opaque
+data:
+  JICOFO_AUTH_USER: '{{ .Values.jicofo.xmpp.user | b64enc }}'
+  JICOFO_AUTH_PASSWORD: '{{ default (randAlphaNum 10) .Values.jicofo.xmpp.password | b64enc }}'
+  JICOFO_COMPONENT_SECRET: '{{ default (randAlphaNum 10) .Values.jicofo.xmpp.componentSecret | b64enc }}'
```

Comment:
You specified `component_password` in the `values.yaml`, but you are using `componentSecret` here.
Caused an authentication issue in my cluster because it will use a generated password instead of the defined one. 🙂

Reply:
Indeed, I will fix that ;)

Reply:
And now fixed...